index.html

```diff
@@ -6,11 +6,73 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
+      <body class="body">
 
-<body>
-    <!-- Add your markup here -->
-</body>
-
-</html>
\ No newline at end of file
+          <!--HEADER-->
+          <header class="header">
+            <!--LOGO & HAMBURGER MENU-->
+              <div class="homepage-header">
+                    <div class="logo-img"> <img  class="logo-img" src ="images/arno-senoner-MRjjcDIk3Gw-unsplash.jpg" alt ="logo-img"/></div>
+                  <div class="hamburger-logo"><img class="hamburger-logo" src="images/threelines.png" alt="hamburger-menu"/> </div>
+              </div>
+               <div class="header-p">
+                    <p>The best cakes in town delivered to your door!</p>
+               </div>
+            </header>
+           <!---------MAIN--------->
+         <main class= "main">
+            <nav class="nav-bar">
```

Comment:
In my opinion, nav tag should be part of the header, not the main content. Main tag should only contain the contents of the web page